operator/pkg/utils/iamauthenticator/iamauthenticator.go

```diff
@@ -0,0 +1,149 @@
+/*
```

Comment:
operator/pkg/components/iamauthenticator.go

iamauthenticator.Pod(...) v1.PodTemplateSpec

In general, I think this is a decent pattern for all of our components (kcm/kcp/ksh).